packages/components/src/tabs/types.ts

```diff
@@ -139,4 +139,12 @@ export type TabPanelProps = {
 	 * Custom CSS styles for the rendered `TabPanel` component.
 	 */
 	style?: React.CSSProperties;
+	/**
+	 * Determines whether or not the tabpanel element should be focusable.
```

Comment:
This is very minor, but I just noticed the variety of formatting of `TabPanel` in the types. 

<img width="886" alt="Screenshot 2023-11-06 at 11 10 06 AM" src="https://github.com/WordPress/gutenberg/assets/35543432/5429d2b6-caca-4733-91a9-1389bff8d836">

Reply:
Good catch. I made one small change, but I didn't update the `style` JSDoc because that one's going away in a separate PR merging soon, and changing it here would have just been asking for a merge conflict to reconcile 😆